dump/main.py

```diff
@@ -170,10 +176,12 @@ def populate_fv(info, module, namespace, conn_pool):
             for key in info[id][db_name]["keys"]:
                 if db_name == "CONFIG_FILE":
                     fv = db_cfg_file.get(db_name, key)
+                elif dash_object and db_name == "APPL_DB":
+                    pb_data = redis_conn.hgetall(key)
+                    fv = get_decoded_value(dash_object, pb_data)
```

Comment:
>get_decoded_value [](http://example.com/codeflow?start=25&length=17)

you can `import` immediate before this line, and no need to swallow any exception.